tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/xml/XMLParserNegativeTest.java

```diff
@@ -36,22 +36,23 @@ public void testXmlArg(String xmlValue, String expectedErrorMessage) {
             Assert.fail("Negative test failed for: `" + xmlValue + "'. Expected exception with message: " +
                     expectedErrorMessage);
         } catch (Exception e) {
-            Assert.assertEquals(e.getMessage(), "failed to parse xml: ParseError at [row,col]:" + expectedErrorMessage);
+            Assert.assertEquals(e.getMessage(), "failed to parse xml: " + expectedErrorMessage);
         }
     }
 
     @DataProvider(name = "xmlValues")
     public Object[][] xmlValues() {
         return new Object[][]{
-                {"<book name=\"irshad<\"></book>" , "[1,19]\nMessage: The value of attribute \"name\" associated " +
-                        "with an element type \"book\" must not contain the '<' character."},
-                {"<!-- comments cannot have -- in it -->" , "[1,29]\nMessage: The string \"--\" is not permitted " +
-                        "within comments."},
-                {"<-note>irshad</-note>", "[1,2]\nMessage: The markup in the document preceding the root element " +
-                        "must be well-formed."},
-                {"<note><.b>irshad</.b></note>", "[1,8]\nMessage: The content of elements must consist of " +
-                        "well-formed character data or markup."},
-                {"<?pi cannot contain ?> in it?><book></book>", "[1,24]\nMessage: Content is not allowed in prolog."}
+                {"<book name=\"irshad<\"></book>" , "Unexpected '<'  in attribute value\n" +
+                        " at [row,col {unknown-source}]: [1,19]"},
+                {"<!-- comments cannot have -- in it -->" , "[com.ctc.wstx.exc.WstxLazyException] String '--' not " +
```

Comment:
Lets create an issue to remove com.ctc.wstx.exc.WstxLazyException from message

Reply:
Created https://github.com/ballerina-platform/ballerina-lang/issues/40127